recipes/nuraft/all/conanfile.py

```diff
@@ -68,6 +73,14 @@ def build(self):
         cmake = CMake(self)
         cmake.configure()
         cmake.build()
+        rmdir(self, os.path.join(self.package_folder, "lib", "cmake"))
+        if self.options.shared:
```

Comment:
Version 3.x generate both static and shared libraries: https://github.com/eBay/NuRaft/blob/v3.0.0/CMakeLists.txt#L338